### PR TITLE
Fix text color for Compose theme in night mode

### LINKED ISSUE #17085: Compose theme not applying correct text color in night mode
### Expected behavior
In night mode, the text color should be white, since the background color is a dark gray.

### Actual behavior
The text color is pitch black, on a dark gray background.

|  |
| --- |

### Steps to reproduce the behavior
0. Make sure the device is set to night mode
1. Open Jetpack app and Login if you're not already logged in
3. Tap `Me`
4. Tap `Scan Login Code`
5. In a guest Chrome session on a computer, go to www.wordpress.com and 
   - Tap `Log In`
   - Tap `Login via the mobile app`
6. Scan the QR Code with your device
7. **Notice** the title is colored pitch black, and the background is dark gray → making the text barely readable.

### PR DESCRIPTION
**Pull request by ovitrif**
Fixes #17085

> The implementation applies the suggestion from here: https://github.com/wordpress-mobile/WordPress-Android/pull/16934#discussion_r929884451.

To test:

0. Make sure the device is set to night mode
1. Open Jetpack app and Login if you're not already logged in
3. Tap `Me`
4. Tap `Scan Login Code`
5. In a guest Chrome session on a computer, go to www.wordpress.com and 
   - Tap `Log In`
   - Tap `Login via the mobile app`
6. Scan the QR Code with your device
7. **Expect** the title text to be colored white → making the text readable.

## Regression Notes
1. Potential unintended areas of impact
  Title text color in `day mode` on the screens after scanning the login code.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
  Manual testing & compose preview.

3. What automated tests I added (or what prevented me from doing so)
  No tests warranted by UI tweaks.

### Previews

| Before | After |
| --- | --- |
| d_old | d_new |

<details>
<summary>Day mode unchanged</summary>

|  |
| --- |

No regression here :shipit:.
</details>

PR submission checklist:

- [x] I have completed the Regression Notes.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: WordPress/src/main/java/org/wordpress/android/ui/compose/theme/AppThemeWihoutBackground.kt [hunk 1]
```diff
@@ -11,22 +11,22 @@ import org.wordpress.android.BuildConfig
  * See [Compose Docs](https://developer.android.com/jetpack/compose/themes/material#content-color) for more details.
  */
 @Composable
-fun AppThemeWithBackground(
+fun AppTheme(
     isDarkTheme: Boolean = isSystemInDarkTheme(),
     content: @Composable () -> Unit
 ) {
-    AppTheme(isDarkTheme) {
+    AppThemeWithoutBackground(isDarkTheme) {
```
Comment (mkevins) [inline WordPress/src/main/java/org/wordpress/android/ui/compose/theme/AppThemeWihoutBackground.kt]: Nice 👍 . I like the semantics of this when leaving public the theme without surface. Also, the doc comments are informative!

**Review by mkevins — APPROVED** [review]
Thanks for fixing this Ovi! I tested this out on a Pixel 3a (physical device) and it is working as described. The code changes also look good.

I've added a suggestion commit on this branch:

https://github.com/wordpress-mobile/WordPress-Android/compare/issue/17085-compose-theme-night-mode-text-color...issue/17085-compose-theme-night-mode-text-color--suggestion

to fix a filename typo, and some mint lint issues (also adding a dark-mode preview for the content). If it's useful, feel free to fast forward this branch to that one before merging 👍 (I've tested both).

**Comment by mkevins** [comment]
Oh, a reminder to add the release note, too, since this is already in production.

**Comment by ovitrif** [comment]
> I've added a suggestion commit on this branch:
> 
> [issue/17085-compose-theme-night-mode-text-color...issue/17085-compose-theme-night-mode-text-color--suggestion](https://github.com/wordpress-mobile/WordPress-Android/compare/issue/17085-compose-theme-night-mode-text-color...issue/17085-compose-theme-night-mode-text-color--suggestion)
> 
> to fix a filename typo, and some mint lint issues (also adding a dark-mode preview for the content). If it's useful, feel free to fast forward this branch to that one before merging 👍 (I've tested both).

Thank you for this @mkevins, that's really awesome ❤️

I added your suggestions to the branch of this PR 🎉

File: WordPress/src/main/java/org/wordpress/android/ui/compose/theme/AppThemeWihoutBackground.kt (hunk 1 above)
Comment (ovitrif) [reply WordPress/src/main/java/org/wordpress/android/ui/compose/theme/AppThemeWihoutBackground.kt]: Team work 🎸 🚀

**Pull request merged**